rules-java/api/src/main/resources/reports/templates/ignored_files.ftl

```diff
@@ -35,23 +35,25 @@
 
 
 <#macro fileRegexesRenderer reportModel>
-    <div class="panel panel-primary">
-        <table class="table table-striped table-bordered">
-            <tr>
-                <th>Regex</th>
-                <th>Compilable</th>
-            </tr>
+    <#if reportModel.fileRegexes.iterator()?has_content>
+        <div class="panel panel-primary">
+            <table class="table table-striped table-bordered">
+                <tr>
+                    <th>Regex</th>
+                    <th>Compilable</th>
+                </tr>
 
-            <#list reportModel.fileRegexes.iterator() as regex>
-            <tr>
-                <td> <#if regex.regex?has_content> ${regex.regex} </#if> </td>
-                <td> <#if regex.compilationError?has_content> ${regex.compilationError}
-                     <#else> OK
-                     </#if> </td>
-            </tr>
-            </#list>
-        </table>
-    </div>
+                <#list reportModel.fileRegexes.iterator() as regex>
+                <tr>
+                    <td> <#if regex.regex?has_content> ${regex.regex} </#if> </td>
+                    <td> <#if regex.compilationError?has_content> ${regex.compilationError}
+                         <#else> OK
+                         </#if> </td>
```

Comment:
I also went ahead and changed it to your preferred format. That is clearly better. :)